external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -1933,7 +1934,8 @@ abstract class AvroSuite
       // By default we should fail to read ancient datetime values when parquet files don't
       // contain Spark version.
       "2_4_5" -> failInRead _,
-      "2_4_6" -> successInRead _
+      "2_4_6" -> successInRead _,
+      "3_2_2" -> successInRead _
```

Comment:
3.2.2? We have not released 3.2.1 yet.

Reply:
I used `branch-3.2` to generate the golden files. See the version in PR description:
```
$ java -jar parquet-tools-1.12.0.jar meta sql/core/src/test/resources/test-data/before_1582_timestamp_micros_v3_2_2.snappy.parquet
file:        file:/Users/maximgekk/proj/parquet-rebase-save-tz/sql/core/src/test/resources/test-data/before_1582_timestamp_micros_v3_2_2.snappy.parquet
creator:     parquet-mr version 1.12.2 (build 77e30c8093386ec52c3cfa6c34b7ef3321322c94)
extra:       org.apache.spark.version = 3.2.2
```

Reply:
I have checked out `branch-3.2` and ran:
```
$ ./build/mvn -Phive -Phive-thriftserver -Dskip -DskipTests package
```
and got `core/target/extra-resources/spark-version-info.properties`:
```
$ cat ./core/target/extra-resources/spark-version-info.properties
version=3.2.2-SNAPSHOT
user=maximgekk
revision=d1cd110c20817eb1ccd716e099be5712df1f670c
branch=branch-3.2
date=2021-12-23T19:12:40Z
url=https://github.com/apache/spark.git
```
should be **3.2.1-SNAPSHOT** Not? cc @gengliangwang @cloud-fan ?

Reply:
Let me take the commit https://github.com/apache/spark/commit/5d45a415f3a29898d92380380cfd82bfc7f579ea and generate the golden files using it.

Reply:
Yes it should be 3.2.1-SNAPSHOT..The RC script aborted sometimes. This should be a mistake when I rerun the script.

Reply:
@gengliangwang can we fix it? otherwise, we will have a problem when releasing 3.2.1.